thrift0.11/build.gradle

```diff
@@ -44,5 +44,7 @@ ext {
 }
 
 // Disable checkstyle because it's checked by ':thrift'.
-tasks.checkstyleMain.onlyIf { false }
-tasks.checkstyleTest.onlyIf { false }
+if (!project.hasProperty('noCheckstyle')) {
```

Comment:
Think it's better to guard with `plugins.withId('checkstyle')`

Reply:
Realized that `Checkstyle` is a core task class which is always available, so I just used `tasks.withType(Checkstyle)` because it is safer.